.github/workflows/ci.yaml

```diff
@@ -194,3 +194,24 @@ jobs:
         with:
           name: install-c9s.tar
           path: install.tar
+  # Doc: https://github.com/redhat-plumbers-in-action/differential-shellcheck#usage
+  differential-shellcheck:
+    name: Differential ShellCheck
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      security-events: write
+      pull-requests: write
```

Comment:
Why does it need the ability to write to the pull request?  Is that for the inline markup comments?

Reply:
Yes, `pull-requests: write` is for comments. It can be removed, and defects will still be marked in the `Files changed` tab. Also, when there are many defects comments aren't created (to keep the conversation "clean").

Reply:
I personally find bots auto-commenting on the pull requests to be more noise than signal.  So let's please remove these permissions.